rust/worker/src/assignment/rendezvous_hash.rs

```diff
@@ -10,13 +10,13 @@ use thiserror::Error;
 use murmur3::murmur3_x64_128;
 
 /// A trait for hashing a member and a key to a score.
-trait Hasher {
+pub(crate) trait Hasher {
```

Comment:
Note to self: `pub` exports this trait from the crate. `pub(crate)` makes it usable anywhere _within_ this crate.